src/main/java/com/alipay/remoting/util/RemotingUtil.java

```diff
@@ -193,16 +193,23 @@ private static String doParse(String addr) {
         if (StringUtils.isBlank(addr)) {
             return StringUtils.EMPTY;
         }
-        if (addr.charAt(0) == '/') {
-            return addr.substring(1);
-        } else {
-            int len = addr.length();
-            for (int i = 1; i < len; ++i) {
-                if (addr.charAt(i) == '/') {
-                    return addr.substring(i + 1);
+        // Find the first and second '/' positions
```

Comment:
What is the relationship between this change and 'support jdk 17&21 workflow' ?

Reply:
In higher versions of JDK, calling SocketAddress.toString() may result in an /unresolved address. This needs to be removed. Additionally, when there is a local proxy, it may appear as /127.0.0.1/proxy:port. The correct IP address and port need to be extracted, so this needs to be handled accordingly.
```
RemotingUtilTest.testParseSocketAddressToString_MuiltiFormatTest:180 expected:<127.0.0.1[]:1111> but was:<127.0.0.1[/<unresolved>]:1111> 
```

Reply:
Get it